### PR TITLE
Authentication tests and Fetch stories test

### PR DESCRIPTION
**Pull request by Tushar19varshney**
Authentication i.e Login and Register of user tests and fetching stories tests added.
I have used `unittest` package of python for testing. 
#104

### CODE REVIEW

File: fact-bounty-flask/app.py [hunk 1]
```diff
@@ -38,20 +38,25 @@ def test(coverage):
         sys.exit(subprocess.call(sys.argv))
     
     import unittest
-    tests = unittest.TestLoader().discover('tests')
-    unittest.TextTestRunner(verbosity=2).run(tests)
-    if COV:
```
Comment (Anmolbansal1) [inline fact-bounty-flask/app.py]: Include COV, it will help for test coverage

File: fact-bounty-flask/app.py [hunk 2]
```diff
@@ -38,20 +38,25 @@ def test(coverage):
...
+            # else, just load all the test cases from the module.
+            suite.addTest(unittest.defaultTestLoader.loadTestsFromName(t))
+
+    unittest.TextTestRunner().run(suite)
 
 @app.cli.command()
 def deploy():
     # migrate database to latest revision
-    upgrade()
+    upgrade()
```
Comment (Anmolbansal1) [inline fact-bounty-flask/app.py]: Insert new line at end of file

File: fact-bounty-flask/app.py (hunk 1 above)
Comment (Tushar19varshney) [reply fact-bounty-flask/app.py]: Test coverage was not working when I executed the test.
Let me figure that out!

File: fact-bounty-flask/app.py (hunk 2 above)
Comment (Tushar19varshney) [reply fact-bounty-flask/app.py]: Ohk!

**Comment by ivantha** [comment]
Closes #104

**Comment by ivantha** [comment]
@Tushar19varshney @Anmolbansal1 Please take a look at #145 if you have time.

**Comment by Tushar19varshney** [comment]
@ivantha Sure :smile:

**Pull request merged**